components/library/tntnet/tntnet-server.p5m

```diff
@@ -33,5 +33,5 @@ file path=usr/share/man/man7/tntnet.xml.7
 
 file path=usr/share/tntnet/tntnet-conf2xml.pl
 
-file path=usr/etc/init.d/tntnet
-file path=usr/etc/tntnet/tntnet.xml
+file path=etc/init.d/tntnet
```

Comment:
So far i then can only hope to get back to this by/on the weekend and
fulfull the SMF part ;-)

In practice of a project @work, we generate a webapp project (a .so loaded
by tntnet program) accompanied by an xml configuration file, so the default
service instance is not used anyway...

--
Typos courtesy of GMail on my Samsung Android
23.03.2016 19:55 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/library/tntnet/tntnet-server.p5m
> <https://github.com/OpenIndiana/oi-userland/pull/1777#discussion_r57216955>
> :
>
> > @@ -33,5 +33,5 @@ file path=usr/share/man/man7/tntnet.xml.7
> >
> >  file path=usr/share/tntnet/tntnet-conf2xml.pl
> >
> > -file path=usr/etc/init.d/tntnet
> > -file path=usr/etc/tntnet/tntnet.xml
> > +file path=etc/init.d/tntnet
>
> So, I see two options:
> a) it can wait for SMF to be created;
> b) we don't deliver init.d script (or daemon at all), so that people don't
> start using it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1777/files/524003a8aec6e89d765aee811e1be2f5c9ae3d46..87f86c70d8e66cfc331367a2e3db5c1612774cc5#r57216955>
>